src/core_modules/capture-core/components/WidgetBulkDataEntry/WidgetBulkDataEntry.types.js

```diff
@@ -0,0 +1,15 @@
+// @flow
+
+export type DataStoreConfiguration = {|
+    configKey: string,
+    dataKey?: string,
+    programId: string,
+    title: string,
+    subtitle?: string,
+    pluginSource: string,
+|};
```

Comment:
Can we move this type def to `components/common/bulkDataEntry` as well (see other comment)